Makefile

```diff
@@ -21,18 +21,16 @@ REPO = $(DOCKER_REPO)/kubermatic$(shell [ "$(KUBERMATIC_EDITION)" != "ce" ] && e
 CMD = $(filter-out OWNERS nodeport-proxy kubeletdnat-controller, $(notdir $(wildcard ./cmd/*)))
 GOBUILDFLAGS ?= -v
 GOOS ?= $(shell go env GOOS)
-GITTAG = $(shell git describe --tags --always)
```

Comment:
Fun fact: we had some places that tried to override this variable from the outside, but that never had any effect.